setup.cfg

```diff
@@ -12,6 +12,7 @@ max-line-length = 88
 per-file-ignores =
     pyuvdata/tests/*.py: D
     pyuvdata/*/tests/*.py: D
+    pyuvdata/uvdata/tests/test_mir_parser.py: D, B035
```

Comment:
I don't love this because I think it actually is a sensible check. That being said, this is a test file and I'm not quite sure how to fix it immediately. Maybe we can merge this but ask @kartographer to take a look and see if he thinks ignoring it here is the best answer or if there's a way to write it that is equally good and doesn't trigger this check. The worry is that you're ignoring the whole file here, not just those lines.

Reply:
dropped the changes here for karto's actual fix in the rebase